### DISCUSSION TITLE
Open Dataverse webresources in VS Code

### DISCUSSION THREAD

**Discussion by magnusjeppesen** [Power-Maverick/DataverseDevTools-VSCode#121] [Q&A] [answered]
Hi

Is it possible to open and edit fx. javascript webresources that is already in Dataverse?
I.e. not creating a new local file and linking that, but edit an existing Dataverse webresource.

Kind regards
Magnus

**Comment by Power-Maverick** [comment, marked as answer]
Currently, there is no download of the resources from Dataverse. You can log a feature request.
